lib/code_fund_web/controllers/campaign_controller.ex

```diff
@@ -67,129 +67,149 @@ defmodule CodeFundWeb.CampaignController do
-        status: [
-          type: :select,
-          label: "Status",
-          opts: [choices: CodeFund.Campaigns.statuses()]
-        ],
-        active_dates: [
-          type: :text_input,
-          label: "Date Range",
-          opts: [
-            "data-target": "date-range.datePicker"
-          ]
-        ],
-        creative_id: [
-          type: :select,
-          label: "Creative",
-          opts: [
-            prompt: "Select a Creative",
-            choices: creatives,
-            "data-target": "campaign-form.creatives"
-          ]
-        ],
-        audience_id: [type: :select, label: "Audience", opts: [choices: audiences]],
-        user_id: [
-          type: :select,
-          label: "User",
-          opts: [
-            prompt: "Select a User",
-            choices:
-              CodeFund.Users.get_by_role("sponsor")
-              |> FormHelpers.repo_objects_to_options([:first_name, :last_name], " "),
-            "data-target": "campaign-form.userId",
-            "data-action": "change->campaign-form#creativesForUser"
-          ]
-        ],
-        ecpm: [
-          type: :currency_input,
-          label: "eCPM",
-          opts: [
-            step: "0.01",
-            min: "0",
-            "data-target": "campaign-form.ecpm",
-            "data-action": "change->campaign-form#calculateImpressions",
-            step: "0.01"
-          ]
-        ],
-        included_countries: [
-          type: :multiple_select,
-          label: "Included Countries",
-          opts: [
-            data: [
-              target: "campaign-form.includedCountries",
-              action: "change->campaign-form#generateEstimates",
-              key: "included_countries"
-            ],
-            class: "form-control campaign-form selectize",
-            choices: Framework.Geolocation.Countries.list(),
-            hint: ~s"""
-            <div>
-              <span class="click-option" data-action='click->campaign-form#selectRecommendedCountries'>Recommended</span> |
-              <span class="click-option" data-action='click->campaign-form#selectPopularCountries'>Popular</span> |
-              <span class="click-option" data-action='click->campaign-form#selectAllCountries'>All</span> |
-              <span class="click-option" data-action='click->campaign-form#selectNoCountries'>None</span>
-            </div>
-            """
-          ]
-        ],
-        budget_daily_amount: [
-          type: :currency_input,
-          label: "Daily Max Spend",
-          opts: [step: "0.01"]
-        ],
-        total_spend: [
-          type: :currency_input,
-          label: "Total Spend",
-          opts: [
-            "data-target": "campaign-form.totalBudget",
-            "data-action": "change->campaign-form#calculateImpressions",
-            step: "0.01"
+    fields = [
+      user_id: [
+        type: :select,
+        label: "Advertiser",
+        opts: [
+          disabled: !is_admin,
```

Comment:
Even if this field is disabled, our advertisers are going to know other advertisers by viewing the source right?